docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -1371,58 +1371,185 @@ Billing Engine
 
 PacketFence integrates the ability to use a payment gateway to bill users to gain access to the network. When configured, the user who wants to access the network / Internet is prompted by a page asking for it's personnal information as well as it's credit card information.
 
-PacketFence currently supports two payment gateways: Authorize.net and Mirapay.
+PacketFence currently supports four payment gateways: Authorize.net, Mirapay, Paypal and Stripe.
 
-The configuration to use the feature is fairly simple. The general configuration to enable / disable the billing engine can be done through the Web administration GUI (*Configuration -> Portal Profiles and Pages*) or from the `conf/profiles.conf` file:
+In order to activate the billing, you will need to configure the following components : 
 
-  [default]
-  billing_engine = enabled
-  ...
+* Billing source(s)
+* Billing tier(s)
+* Portal profile
 
-Billing engine parameters are specified in `conf/pf.conf` or from *Configuration -> Billing*:
+Configuring a billing source
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-  [billing]
-  gateway = authorize_net
-  authorizenet_posturl = The payment gateway processing URL
-  authorizenet_login = The merchant's unique API Login ID
-  authorizenet_trankey = The merchant's unique Transaction Key
+First select a billing provider and follow the instructions below.
+
+Paypal
+++++++
+
+NOTE: This provider requires that your PacketFence server is accessible on the public domain. For this your PacketFence portal should be available on a public IP using the DNS server name configured in PacketFence.
+
+If you have a business account and do not want to configure a test environment, you can skip the next section.
+
+*Sandbox account*
+
+To configure a sandbox paypal account for use in PacketFence, head to https://developer.paypal.com/ and either sign up or login into your existing account.
+
+Then in the Sandbox menu, click 'Accounts'
+
+Create an account that has the type 'Personal' and one that has the type 'Business'.
+
+Afterwards, go back into accounts, and expand the business account, then click 'Profile'
+
+## accounts screen
+
+Now click the 'Change password' link and change the password and note it.
+
+## change password screen
+
+Do the same thing with the personal account you created
+
+*Configuring the merchant account*
+
+Login into the Paypal business account that you created at https://www.sandbox.paypal.com/ if you are using a sandbox account or on https://www.paypal.com/ if you are using a real account.
+
+Next go in 'My Account -> Profile' in order to go into your profile configuration.
+
+Next in the 'Selling Preferences' you will need to select 'Website Payment Preferences'
```

Comment:
On my side the column is named Selling Preferences

You and your European paypal...